### PR TITLE
Fix and disable test cases

### PR DESCRIPTION
**Pull request by anupama-pathirage**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/json/BJSONValueTest.java [hunk 1]
```diff
@@ -410,7 +410,7 @@ public void testJsonArrayToJsonCasting() {
         Assert.assertEquals(returns[0].stringValue(), "[[1, 2, 3], [3, 4, 5], [7, 8, 9]]");
     }
 
-    @Test
+    @Test(groups = {"broken"})
```
Comment (pubudu91) [inline tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/json/BJSONValueTest.java]: What happened with this test?

File: tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/map/ConstrainedMapTest.java [hunk 2]
```diff
@@ -429,7 +429,8 @@ public void testMapOfElementTypeRefArray() {
         Assert.assertEquals(((BInteger) returns[1]).intValue(), 25);
     }
 
-    @Test(description = "Test json to struct conversion where struct is included with constrained map.")
+    @Test(description = "Test json to struct conversion where struct is included with constrained map.",
+          groups = { "broken" })
```
Comment (pubudu91) [inline tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/map/ConstrainedMapTest.java]: This was fixed

Comment (pubudu91) [reply tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/map/ConstrainedMapTest.java]: in PR #12203

File: tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/json/BJSONValueTest.java (hunk 1 above)
Comment (anupama-pathirage) [reply tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/json/BJSONValueTest.java]: org.ballerinalang.util.exceptions.BLangRuntimeException: error: assertion error: expected 'json[][]', found 'json[]' {}
	at testJsonToJsonArrayCasting(json-value.bal:321)

File: tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/map/ConstrainedMapTest.java (hunk 2 above)
Comment (anupama-pathirage) [reply tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/map/ConstrainedMapTest.java]: Enabled with bdfd7260308b8c3029eb84690680dd0c47566094

**Pull request merged**